README.md

```diff
@@ -23,6 +23,33 @@ bin/dev
 
 Then browse to http://localhost:3000 to see the running application.
 
+#### Change default user role
+
+You can change your abilities within the app by setting the `ROLES` environment
+variable. By default you will be granted administrative abilities.
+
+Note that you may need to clear your browser session cookies to pick up the new
```

Comment:
Can we say less here, like just the first sentence?

Reply:
If you can believe it, this is already trimmed down from what's in H2, IIRC. I agree that it's still verbose. Will have another look.